pytravharv/WebAccess.py

```diff
@@ -88,8 +88,14 @@ def download_uri_to_store(uri, triplestore, format="json-ld"):
             format = "json-ld"
         elif "text/turtle" in r.headers["Content-Type"]:
             format = "turtle"
-        triplestore.parse(data=r.text, format=format, publicID=uri)
-        log.info(f"content of {uri} added to the triplestore")
+        try:
+            triplestore.parse(data=r.text, format=format, publicID=uri)
```

Comment:
public_id

Reply:
here too, if `triplestore` would just be `g`or `graph` it would be more clear where the `parse`method is coming from

i.e. https://rdflib.readthedocs.io/en/stable/_modules/rdflib/graph.html#Graph.parse

so, it would likely more naturally show to @bulricht that `publicId` is not our decision here ;)